monitor-servicehealth-alert/azuredeploy.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "actionGroupName": {
+      "value": "serviceHealthActionGroup"
```

Comment:
Do action group names/shortnames/activityLogAlertNames need to be unique across RGs? If not, this is probably fine; if they do/should, then using GEN-UNIQUE is better.